tests/tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>

```

Comment:
OMG

Reply:
Actually that was leftover from an experiment and now does nothing.  But yeah you can specify multiple frameworks as targets.  Our problem was that CI is using 2.0 and so can't build for 3.1 and I was 3.1 and need to just make some local workspace-esque tweaks and keep them to myself.